provider/neutron/validation.py

```diff
@@ -250,10 +250,11 @@ def subnet_is_ovirt_managed(subnet):
 
 def no_default_gateway_in_routes(default_gateway_exists, routes):
     if default_gateway_exists and routes:
+        destination = RouterMapper.REST_ROUTER_DESTINATION
```

Comment:
One small nit I have noticed, this can be outside the loop.

Reply:
Do you mean outside the 'if'?

Reply:
Yes